client/src/Components/VideoCard.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+const VideoCard = ({ video, removeVideo }) => {
+  const [rating, setRating] = useState(video.rating);
+
+  const addVote = () => {
+    setRating(rating + 1);
+    console.log(rating);
```

Comment:
Would be good not to push `console.log` into remote! remember other devs wouldn't need to see lines you added for debugging.